src/Swashbuckle.AspNetCore.SwaggerGen/Generator/SwaggerGenerator.cs

```diff
@@ -25,20 +25,23 @@ public class SwaggerGenerator : ISwaggerProvider
         }
 
         public SwaggerDocument GetSwagger(
-            string documentName,
+            string documentVersion,
```

Comment:
documentVersion is too specific. Versioning is just one reason why you would define multiple documents but there may be others